### ISSUE TITLE
Header structure skips levels

### ISSUE THREAD

**Issue by chlane** [vmware/clarity#3166]
## Description  
Need better header structure for changelog pages. Implement H1-H4 (as appropriate)  for breaking changes, bugs, released, npm packages, sketch template, git change, and each individual bug.  Don't  skip from H2 to H6 as is being done with bug fixes.  Only one header is required per line (for  example, the individual bugs have a H2 for the description and then it repeats  several times for each of the other elements on that line)

## Location     
Changelog view

## Priority    
 medium

## Recommended fix      
Ensure that the headings levels are not skipped. It is not necessary to use all 6 levels, but there should be no H6 headings if there are no H1, H2, H3, H4 and H5 headings on the page.

## LOE  
medium

**Comment by gnomeontherun** [comment]
No longer an issue or resolved in the upcoming website work. Closing as resolved.
